src/GeositeFramework/plugins/layer_selector_v2/main.js

```diff
@@ -110,16 +108,37 @@ define([
                 return layerId;
             },
 
-            createLayerMenu: function(layerId) {
+            showLayerMenu: function(el) {
+                var $el = $(el),
+                    layerId = this.getClosestLayerId(el),
+                    $menu = this._createLayerMenu(layerId),
+                    $shadow = this._createLayerMenuShadow(),
+                    supportsOpacity = this.state.serviceSupportsOpacity(layerId),
+                    pos = $el.offset(),
+                    top = supportsOpacity ? pos.top : pos.top + 55;
```

Comment:
Not sure why, but the layer tools dissappear when opening the layer menu if the layer is not selected. We could fix this as part of #545.

![image](https://cloud.githubusercontent.com/assets/43062/13081939/b64f9e08-d49c-11e5-9dbe-32a693d1b963.png)